assets/vhs/Dockerfile

```
@@ -0,0 +1,82 @@
+EOF
+
+RUN cat >/home/demo/capsule/README.md <<'EOF'
+# capsule demo
+EOF
+
+RUN chown -R demo:demo /home/demo
+
+USER demo
+WORKDIR /home/demo
+
+RUN git init --quiet --initial-branch=render-cache /home/demo/capsule \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)